test/lib/topology_builder.hh

```diff
@@ -40,6 +40,49 @@ struct shared_load_stats {
 
     void set_capacity(locator::host_id host, size_t capacity) {
         stats.capacity[host] = capacity;
+        stats.tablet_stats[host].effective_capacity = capacity;
+    }
+
+    void set_tablet_size(locator::host_id host, const locator::range_based_tablet_id& rb_tid, uint64_t tablet_size) {
+        stats.tablet_stats[host].tablet_sizes[rb_tid.table][rb_tid.range] = tablet_size;
+    }
+
+    void migrate_tablet_size(locator::host_id src_host, locator::host_id dst_host, const locator::range_based_tablet_id& rb_tid) {
```

Comment:
Can we use load_stats::migrate_tablet_size() for this? It seems to do the same thing.